### ISSUE TITLE
Windows Theme

### ISSUE THREAD

**Issue by murliatdure** [nolimits4web/Framework7#926]
Can we introduce the Windows Theme similar to ionic2.3?

![image](https://cloud.githubusercontent.com/assets/7137859/14373133/97298d86-fd64-11e5-9220-bf1689e810a7.png)

![image](https://cloud.githubusercontent.com/assets/7137859/14373150/b9b83078-fd64-11e5-8cfc-f04e4eab21bc.png)

![image](https://cloud.githubusercontent.com/assets/7137859/14373156/c5eb94d4-fd64-11e5-980f-7514312af364.png)

![image](https://cloud.githubusercontent.com/assets/7137859/14373164/d03c0f5e-fd64-11e5-8786-cff3538a00e7.png)

**Comment by J05HI** [comment]
@murliatdure 
The window market isn't very big. If you like you can make a pull request. 

@nolimits4web @ZanderBrown @valnub 
Should be closed.

**Comment by ZanderBrown** [comment]
The Windows Mobile market maybe small for phones but there is a quite sizable Windows Tablets market.

I'm going to keep this open as a possible feature for 2.0

**Comment by J05HI** [comment]
@ZanderBrown 
Okay, let's see :)

**Comment by valnub** [comment]
As far as I remember there was Windows theme on the "roadmap" but it was so far down on the priority list, I am sure it will never happen unless Windows Phone wins 50% of the market.

**Comment by valnub** [comment]
Windows 10 theme is on the todo-list for v2 or Framework7. However, it is rather low priority.

**Comment by dawuwu** [comment]
Windows Phone marketshare is less than 1% and even most big companies doesnt support it anymore. Would be unnecessary workload to implement this in F7 imho.

**Comment by J05HI** [comment]
@dawuwu 
I'm of your opinion.

**Comment by J05HI** [comment]
@murliatdure 
If you like you can make the theme by your own and create a pull request ;)

**Comment by valnub** [comment]
Windows Phone has no relevance for sure, but providing a Windows **_10_** theme would be great so F7 can be used to build tablet and desktop apps too.

**Comment by ZanderBrown** [comment]
And with the new UWP/Fluent/Whatever Buzzword style Windows Phone and Windows Tablet/Desktop follow the same design patterns

**Comment by dawuwu** [comment]
If F7 supports desktop apps, what about MacOSX and Linux?

**Comment by valnub** [comment]
@dawuwu I don't think that's on the roadmap ;-)
Win10 could be interesting though since it can also run on a phone.

**Comment by J05HI** [comment]
@dawuwu @valnub 
I think Framework7 is a mobile & tablet only framework. For Desktop apps you should use another framework.

**Comment by valnub** [comment]
Well, Win10 can be used on mobile & tablet

**Comment by J05HI** [comment]
@valnub 
Yeah but very less percent are windows devices. 
Low priority is good enough :)

**Comment by krosoftware** [comment]
Why not create single framework for mobile,tablet and desktop?
It can not be bad.
Btw, I'm not aware of anything like that.

**Comment by J05HI** [comment]
@krosoftware 
Yeah would be great but I think there are better designs for desktops.

**Comment by ZanderBrown** [comment]
@J05HI where are you getting these figures from, Windows occupies about a fifth of the Tablet market.

Ans remembering older issues many people have been targeting Windows.

**Comment by J05HI** [comment]
@ZanderBrown 

![](http://mspoweruser.com/wp-content/uploads/2011/04/smartphone-marketshare.png)

**Comment by hashg** [comment]
hmm... March 2011 infographics. That's 6 years old.

**Comment by ZanderBrown** [comment]
... and for smartphones ....

**Comment by krosoftware** [comment]
@J05HI Yes I know that there are better frameworks for desktop but you need to write your software twice.
What I meant is something like this (http://guillaumebiton.github.io/HackerNews7/) where you can put on left column some accordion - this is written i written in Framework7 - with some additional features, like table with priority columns (like here: http://demos.jquerymobile.com/1.4.5/table-column-toggle/) - so the columns are differently displayed on different devices..

**Comment by J05HI** [comment]
@krosoftware 
Yeah, now I understand what you mean.
Would be amazing for iPad too!!!

**Comment by D-Marc1** [comment]
I'd say because of the low overall market share, but modest size in tablet (though tablet is a shrinking market in itself), I think it would nevertheless be better to focus on iOS and Android features. The risk/reward ratio in terms of workload and benefit is just too low for this to be seriously implemented at this point. Now adding some more cool features for iOS/Android or improving some of its existing features is something that would continue to progress this framework, and ultimately be in its best interest I think.

Maybe in the far future, if Framework7 decides to expand to desktop apps for Election. This would be neat for it to be responsive on other platforms, like MacOS, Windows and some Linux distros. But only then could I see this truly paying off for its time spent. Just my two cents

**Comment by neogetz** [comment]
> Maybe in the far future, if Framework7 decides to expand to desktop apps for Election.

You mean Electron I think...

**Comment by RedSparr0w** [comment]
> Windows Phone has no relevance for sure, but providing a Windows 10 theme would be great so F7 can be used to build tablet and desktop apps too.

This is what i would be using it for.
I've already used the material theme for a couple of nodejs electron/nwjs desktop apps.
Would be cool to have a windows theme as well.

**Comment by J05HI** [comment]
@RedSparr0w
Ooooh now you got me.
It would be amazing for Electron apps!
Hope @nolimits4web has time to implement it if he has completed the v2.

**Comment by JosephShenton** [comment]
Something like this? https://github.com/SniperGER/MetroUI

**Comment by RedSparr0w** [comment]
[winstrap](https://github.com/winjs/winstrap) & [MetroUI](https://github.com/SniperGER/MetroUI) both have pretty good windows themes

**Comment by elijnzaad** [comment]
We have a tablet app for iOS and Android. One of our customers wants to standardise on Windows (for whatever reason), so they've asked us to create the same app for use on Surface tablets. That's not a big task, as Cordova supports building Windows apps.
For now we use a modified Android theme with F7, but it would be great if you could supply a Windows theme as well. 

I'm not sure how big the market for apps on Surface tablets will be. But we have had more requests for Surface apps as some of the bigger companies who have centred their infrastructure and technical support around Windows PC's seem to be reluctant to supply their workforce with tablets using 'alien' operating systems like iOS and Android.

So for us it does make some sense to release our app on Windows 10 tablets as well.

**Comment by nolimits4web** [comment]
This issue was incorrectly closed on 2026-05-19 by a compromised credential (Shai-Hulud npm supply-chain worm). Reopening — apologies for the noise.
